src/CLU/Microsoft.CLU.Run/PackageManagementMode.cs

```diff
@@ -215,6 +218,46 @@ private void Install(string packageName, string version, string caption = "Insta
             CLUEnvironment.Console.WriteLine(caption, packageName);
 
             _manager.InstallPackage(package, false, false);
+
+            BuildIndexes(this._packagesRootPath, version, packageName);
+        }
+
+        private void BuildIndexes(string packagesPath, string version, string packageName)
+        {
+            // When exe names don't match their package name, there is no way to find the exe to build indexes
+            // (since executables don't have extenstion on non-Windows).
+            // It is recommended that the package and executable name match.  If they do not, you must include a
+            // BuildIndex.cmd and BuildIndex.sh file so the package can be installed successfully on each platform.
+            const string buildIndexCommandName = "BuildIndex";
+
+            // Only build indexes if the package matches the current runtime
+            string currentRuntime = Platform.GetCurrentRuntime();
```

Comment:
we need this one since we are installing the OSX/Ubuntu packages in Windows. The install succeeds in laying down the bits in pkgs, but we can't run the buildIndexes part since that require executable invocation